src/main/java/com/rackspace/jenkins_nodepool/NodePool.java

```diff
@@ -499,6 +525,20 @@ public FormValidation doCheckLabelPrefix(@QueryParameter String labelPrefix) {
             }
         }
 
+        public FormValidation doCheckRequestTimeout(@QueryParameter String requestTimeout) {
+            LOG.log(Level.INFO, "doCheckRequestTimeout: {0}", requestTimeout);
+            String error = "Request Timeout must be a whole number greater than zero.";
```

Comment:
One could make this allow an empty value and then just apply a default timeout to make this field optional.